mozilla/bedrock
# [fix bug 1416764] Update Firefox favicon for Quantum 57

## Description
- Updates Firefox favicon behind `firefox57-release` switch
- Removes `media/img/firefox/favicon.png` (orphaned, afaict)

## Issue / Bugzilla link
https://bugzilla.mozilla.org/show_bug.cgi?id=1416764

## Testing
- Make sure new favicon shows correctly. 
- Make sure I didn't miss anything?
